docs/content/concepts/cryptography/transaction-auth/offline-signing.mdx

```diff
@@ -72,4 +72,20 @@ Amount: Full Balance
 Status : Success
 Mutated Objects:
  - ID: <OBJECT_ID> , Owner: Account Address ( <SUI-ADDRESS> )
+```
+
+## Alternative: Sign with Sui Keystore and Execute Transaction
+
+Alternatively, you can use the active key in Sui Keystore to sign and output a Base64-encoded sender signed data with flag `--serialize-signed-transaction`. 
+
+```shell
+sui client transfer-sui --to <SUI-ADDRESS> --sui-coin-object-id <COIN-OBJECT-ID> --gas-budget <GAS-AMOUNT> --serialize-signed-transaction
+```
+
+The console responds with the resulting `<SIGNED_TX_BYTES>` value.
+
+After you obtain the signed transaction bytes, you can submit it using the execute combined signed transaction command. This command takes `--signed-tx-bytes` as the signed transaction bytes to execute (see output of the previous `sui client transfer` command). This executes the signed transaction and returns the certificate and transaction effects if successful.
```

Comment:
```suggestion
After you obtain the signed transaction bytes, you can submit it using the `execute-combined-signed-tx` command. This command takes `--signed-tx-bytes` as the signed transaction bytes to execute (see output of the previous `sui client transfer` command). This executes the signed transaction and returns the certificate and transaction effects if successful.
```
You could also add the human readable version in parentheses after the command if you think it's an issue for the user.